lib/corm/model.rb

```diff
@@ -138,6 +140,43 @@ def self.count
       execute(statements['count']).first['count'].to_i
     end
 
+    ##
+    # Find by keys.
+    # This `find` methods wants to be as flexible as possible.
+    #
+    # Unless a block is given, it returns an `Enumerator`, otherwise it yields
+    # to the block an instance of the found Cassandra entries.
+    #
+    # If no keys as passed as parameter, the methods returns (an Enumerator for)
+    # all the results in the table.
+    #
+    # If one or more keys are given, the **order matters**; since this methods
+    # will craft a CSQL query, the first key **must** be the `partition key`,
+    # while the followings are intended as the `clustering keys`... that can be
+    # more than one.
+    #
+    # If the keys passed as parameter are more than the defined by the table the
+    # query is not valid, it cannot be executed and an error is raised.
+    def self.find(*key_values, &block)
```

Comment:
IMO an hash is the wrong signature here.

The keys cannot be passed in any order, they *have* to be in order, and this is a requirement from cassandra.

[`get`](https://github.com/stefanofontanelli/corm/blob/master/lib/corm/model.rb#L145) has a similar signature to this one: a list of keys. Hash are unordered, at least if we don't want to rely on the ordered of insert which I hope it's not what is being proposed :)
The hash makes sense only for the partition keys.

I'm fine with 
- a single parameter, an `Array` of keys
- the current implementation
- a **generated** method. i.e. if the user defined two partition keys and three cluster keys, we can generate a method `def self.find(part_key1, part_key2, clust_key1=nil, clust_key2=nil, clust_key3=nil)` 
- two parameters `def self.find(partition_keys, clustering_keys=[])`